electro/models.py

```diff
@@ -19,177 +21,196 @@ class BaseModel(Model):
 
     is_active = fields.BooleanField(default=True)
     is_deleted = fields.BooleanField(default=False)
-
     date_deleted = fields.DatetimeField(null=True)
 
-    class Meta:  # pylint: disable=too-few-public-methods
+    class Meta:
         """The metaclass for the base model."""
 
         abstract = True
 
 
-# region Discord Models
-class User(BaseModel):
-    """The model for Discord User."""
+# region Core Models
 
-    id = fields.BigIntField(pk=True)
-    username = fields.CharField(max_length=255)
 
-    discriminator = fields.IntField()
-    avatar = fields.CharField(max_length=255, null=True)
+class PlatformId(Model):
```

Comment:
Why do we call it `PlatformId` and not simply `Platform`? When it's called `PlatformId`, the `*_id` prefixed FK-field would be named `platform_id_id` which IMHO doesn't make much sense.